src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -41,6 +46,16 @@ type Props = {
 } & RouteComponentProps<{alertId: string; orgId: string}, {}>;
 
 export default class DetailsBody extends React.Component<Props> {
+  get metricPreset() {
+    const alertRule = this.props.incident?.alertRule;
+    const aggregate = alertRule?.aggregate;
+    const dataset = alertRule?.dataset ?? Dataset.ERRORS;
+
+    return PRESET_AGGREGATES.find(
+      p => p.validDataset.includes(dataset) && p.match.test(aggregate ?? '')
```

Comment:
could move the `?? ''` up with the create of aggregate

Reply:
I'd prefer to keep it closest to where undefined needs to be coerced into a string, which is what test requires.